### PR TITLE
Display input errors inline

### PR DESCRIPTION
**Pull request by mborsare**
We previously weren't showing any error messages. This adds the message
to display "under the line" on the madlib forms.

uh-oh

### CODE REVIEW

File: app/assets/stylesheets/components/_madlib-statement.scss [hunk 1]
```diff
@@ -16,3 +16,27 @@
...
+.field_with_errors {
+  .selectize-control .selectize-input.items {
+    background-color: tint($required-color, 90%);
+    border-bottom-color: $required-color;
+  }
+
+  .error {
+    display: block;
+
+    @at-root .error {
```
Comment (ryancoughlin) [inline app/assets/stylesheets/components/_madlib-statement.scss]: wat dis?

File: app/assets/stylesheets/components/_madlib-statement.scss [hunk 2]
```diff
@@ -16,3 +16,27 @@
...
+      background: image-url("icon_error.svg") 100% center / 1.35em no-repeat;
+      color: $required-color;
+      display: none;
+      font-size: $small-font-size;
+      padding-right: 2em;
```
Comment (ryancoughlin) [inline app/assets/stylesheets/components/_madlib-statement.scss]: You and ali are king/queen of variables on padding variables.....................should this be a variable?

File: app/assets/javascripts/input_validation.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+$(function(){
+  $(document).on("change", ".field_with_errors", function() {
```
Comment (ryancoughlin) [inline app/assets/javascripts/input_validation.js]: Interesting, does this get triggered when the DOM of `.field_with_errors` changes?

File: app/assets/stylesheets/components/_madlib-statement.scss (hunk 1 above)
Comment (mborsare) [inline app/assets/stylesheets/components/_madlib-statement.scss]: It tells sass, "hey bump this up outside of the nested selector and make a rule." It's a way to keep things a little more organized. So, rather that having a `.error { display: none}` randomly in the sheet. Now, there's a logical relationship.

File: app/assets/javascripts/input_validation.js (hunk 3 above)
Comment (mborsare) [inline app/assets/javascripts/input_validation.js]: I'm pretty sure, yes. @derekprior wrote this. He can confirm.

File: app/assets/stylesheets/components/_madlib-statement.scss (hunk 2 above)
Comment (mborsare) [inline app/assets/stylesheets/components/_madlib-statement.scss]: This is one of those cases where no variable felt right. Much like the type scale conversation we had the other day.

**Pull request merged**